manifests/init.pp

```diff
@@ -73,12 +72,21 @@
     sysctl {
       default:
         before => Class['docker'];
-      'net.bridge.bridge-nf-call-iptables':  value => 1 ;
-      'net.bridge.bridge-nf-call-ip6tables': value => 1 ;
+      'net.bridge.bridge-nf-call-iptables':  value => 1;
+      'net.bridge.bridge-nf-call-ip6tables': value => 1;
     }
   }
 
   class { 'docker':
     * => $_default_options + $_socket_group_option + $options
   }
+
+  if $facts['osfamily'] == 'RedHat' {
+    exec { 'hack around broken puppetlabs/docker':
```

Comment:
This should definitely be in the readme, since it looks like it makes the `docker` group the same as the `dockerroot` group?

Reply:
I put an entry in the README file.

Reply:
@jeefberkey Actually, the downstream module doesn't work any other way and can't be overridden so, while it's great to have it in the README, it honestly doesn't matter in the end anyway because you can't do anything else.